CMakeLists.txt

```diff
@@ -31,34 +31,35 @@ endif()
 # Include the generated CMake config.
 include(openssl.cmake)
```

Comment:
Regarding this inclusion, and the fact that our gentemplate adds the collected build information at the end, I've been thinking of looking of [target_sources](https://cmake.org/cmake/help/v3.17/command/target_sources.html) could help.  This command is, as far as I understand, supposed to be used for exactly the thing that has appeared troublesome: adding source data very late in CMakeLists.txt